{"levelname": "ERROR", "asctime": "2025-04-15T20:05:49.015Z", "name": "root", "message": "Collection collection123 not found", "lineno": 115, "pathname": "/single-cell-data-portal/backend/layers/processing/upload_failures/app.py"}
